CMakeLists.txt

```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```

Comment:
Real question though, have we decided that we are willing to give up VS2013 support yet? And, how is VS2015's C++14 support? Also I wonder which versions of gcc we are willing to support?